app/src/main/java/com/akoscz/youtube/YouTubeRecyclerViewFragment.java

```diff
@@ -55,6 +56,8 @@
     private Spinner mPlaylistSpinner;
     private PlaylistCardAdapter mPlaylistCardAdapter;
     private YouTube mYouTubeDataApi;
+    private ProgressDialog progressBar;
```

Comment:
Please follow the class member naming convention that's present in the file.   Rename to mProgressBar.  Thanks!